### PR TITLE
fix(replays): Updated Replay fast forward copy

### LINKED ISSUE #38476: Add note on the fast forwarding indicator for inactivity that happens in the Replay video
**Problem:**

Users are confused between the fast forward indicator and the playback speed setting.

**Solution:**

Fast forward indicator should say “fast forwarding through inactivity” and not have the "x3" speed displayed.

Screen Shot 2022-08-31 at 4 28 08 PM

### PR DESCRIPTION
**Pull request by danecando**
### Changes
- Changes the `-> {speed}x` copy to `Fast forwarding through inactivity ->`
- Adds the speed to the tooltip `Fast forwarding at {speed}x`

Closes #38476 

Screen Shot 2022-09-14 at 2 58 59 PM

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: static/app/components/replays/player/fastForwardBadge.tsx [hunk 1]
```diff
@@ -13,9 +13,9 @@ type Props = {
 function FastForwardBadge({speed, className}: Props) {
   return (
     <Badge className={className}>
-      <FastForwardTooltip title={t('Fast forwarding')}>
+      <FastForwardTooltip title={t(`Fast forwarding at ${speed}x`)}>
+        {t('Fast forwarding through inactive')}
```
Comment (jas-kas) [inline static/app/components/replays/player/fastForwardBadge.tsx:17]:
```suggestion
        {t('Fast forwarding through inactivity')}
```

**Review by jas-kas — APPROVED** [review]

File: static/app/components/replays/player/fastForwardBadge.tsx (hunk 1 above)
Comment (jas-kas) [reply static/app/components/replays/player/fastForwardBadge.tsx:17]: Slight edit that makes more sense - I realize the GH issue I created missed this. 😄

**Comment by ryan953** [comment]
@dane can you update the description after code changes?

**Comment by dane** [comment]
@ryan953 I second your request for a description update 🙂

**Pull request merged**